src/v7400/object/model.rs

```diff
@@ -57,4 +59,56 @@ impl<'a> ModelHandle<'a> {
                 _ => None,
             })
     }
+
+    /// Gets the root-most model parent of this object in the scene hierarchy.
+    ///
+    /// If this object has no parents, returns this object.
+    ///
+    /// There may be multiple root models in a single scene.
+    pub fn root_model<'b>(&'b self) -> TypedModelHandle<'a> {
+        let mut parent : TypedModelHandle = match self.get_typed() {
+            TypedObjectHandle::Model(o) => o,
+            _ => panic!("ModelHandle should always have TypedObjectHandle::Model type.")
+        };
+
+        while let Some(m) = parent.parent_model() {
+            parent = m;
+        }
+        return parent;
+    }
+
+    /// Returns the local rotation (Lcl Rotation) of this model object, if one is present.
+    pub fn local_rotation(&self) -> Option<Vector3<f64>> {
+        let rotation = self.direct_properties()?.get_property("Lcl Rotation")?;
+
+        // Pull out the first three float attributes.
+        let components: Vec<f64> = rotation.node().attributes().iter().filter_map(|attr| {
+            return match attr {
+                AttributeValue::F64(value) => Some(value.clone()),
+                _ => None
+            }
+        }).collect();
+
+        // todo: Is it ever possible for a rotation to return a quaternion (ie. 4 values)?
+        assert_eq!(components.len(), 3);
+
+        return Some(Vector3::from_slice(components.as_slice()));
+    }
+
+    /// Returns the local scale (Lcl Scale) of this model object, if one is present.
+    pub fn local_scale(&self) -> Option<Vector3<f64>> {
+        let rotation = self.direct_properties()?.get_property("Lcl Scaling")?;
+
+        // Pull out the first three float attributes.
+        let components: Vec<f64> = rotation.node().attributes().iter().filter_map(|attr| {
+            return match attr {
+                AttributeValue::F64(value) => Some(value.clone()),
+                _ => None
+            }
+        }).collect();
+
+        assert_eq!(components.len(), 3);
+
+        return Some(Vector3::from_slice(components.as_slice()));
+    }
```

Comment:
Same as `local_rotation`.

Additionally, I feel the term "local scaling" is preferable, as "scaling" is a term used in a FBX data (same reason as using `local_rotation` rather than `local_rotate`).

```suggestion
    /// Returns the local scaling (Lcl Scaling) of this model object, if one is present.
    pub fn local_scaling(&self) -> Option<PropertyHandle<'_>> {
        // `Model` object has native name `FbxNode`.
        self.properties_by_native_typename("FbxNode")
            .get_property("Lcl Scaling")
    }
```